src/components/page-header/search-popover.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Badge, IconButton, Popover, PopoverBody, PopoverContent, PopoverTrigger } from '@chakra-ui/react';
+import { MdSearch } from 'react-icons/md';
+import { RmgAutoComplete, RmgFields, RmgFieldsField } from '@railmapgen/rmg-components';
+import { StnId } from '../../constants/constants';
+import { useRootDispatch } from '../../redux';
+import { setSvgViewBoxMin, setSvgViewBoxZoom } from '../../redux/param/param-slice';
+import { setSelected } from '../../redux/runtime/runtime-slice';
+import { getCanvasSize, pointerPosToSVGCoord } from '../../util/helpers';
+import { useWindowSize } from '../../util/hooks';
+import { StationAttributes } from '../../constants/stations';
+
+interface SearchData {
+    id: StnId;
+    value: string;
+}
+
+export const SearchPopover = () => {
+    const [isOpen, setIsOpen] = React.useState(false);
+
+    const dispatch = useRootDispatch();
+    const graph = React.useRef(window.graph);
+
+    const size = useWindowSize();
+    const { width, height } = getCanvasSize(size);
+
+    const [data, setData] = React.useState<SearchData[]>([]);
+    React.useEffect(() => {
+        if (isOpen) {
+            setData(
+                graph.current
+                    .nodes()
+                    .filter(node => node.startsWith('stn'))
+                    .map(stnId => {
+                        const attr = graph.current.getNodeAttributes(stnId);
+                        const type = attr.type;
+                        const name = (attr[type] as StationAttributes).names.join('/');
+                        return { id: stnId, value: name } as SearchData;
+                    })
+            );
+        }
+    });
+
+    const handleSearch = (stnId: StnId) => {
```

Comment:
This is not a search but setting the station to the center of the canvas.

Reply:
renamed to moveCanvas